### PR TITLE
Handle home page

### PR DESCRIPTION
**Pull request by RandSohail**
- Fetch data
- Pass proper props to it's component

### CODE REVIEW

File: server/controllers/home/index.js [hunk 1]
```diff
@@ -4,13 +4,14 @@ const { customError } = require('../errors');
 const getHomeData = async (request, response, next) => {
   try {
     const receivedData = await Promise.all([
-      Location.findAll(),
-      MainServices.findAll(),
+      Location.findAll({ attributes: ['city'] }),
```
Comment (devstm) [inline server/controllers/home/index.js:7]: we need the city id, main service id and sub-service id also

File: client/src/Pages/Search.tsx [hunk 2]
```diff
@@ -4,12 +4,13 @@ import { useLocation } from 'react-router-dom';
 
 function Search() {
   const location = useLocation();
-  const searchLocation = location.state;
+  const search : any = location.state;
 
   return (
     <div className="container">
       {useTranslation().t('search-greeting')}
```
Comment (devstm) [inline client/src/Pages/Search.tsx:11]: {useTranslation().t('search-greeting')}
this text must be inside html element to make styling easy 
may be <h1> {useTranslation().t('search-greeting')} </h1> better?

File: client/src/Pages/Home.tsx [hunk 3]
```diff
@@ -9,119 +10,55 @@ import {
...
-        <Divider />
-        <SearchByLocation locationArray={locations} />
-      </div>
-      <JoinUs />
-      <div className="container">
-        <Reviews reviews={reviews} />
-      </div>
+      {isLoading
+        ? (
+          <div className="spinier-loading">
```
Comment (devstm) [inline client/src/Pages/Home.tsx:42]: can you split spinier in new component ?

File: client/src/Components/WelcomeSearch/index.tsx [hunk 4]
```diff
@@ -14,19 +14,25 @@ interface ListOfLocation {
...
   return (
     <div className="basic-container-welcome-home-page">
       <div className="welcome-search-home-right">
         <h2>{t('home-welcome-search')}</h2>
         <Divider className="divider-welcome-home-page" />
         <Input.Group compact>
-          <Input style={{ width: '40%' }} placeholder={t('home-search-name')} size="middle" />
-          <Select defaultValue="بيت حانون" size="middle">
-            {location.map((item) => <Option key={item.city} value={item.city}>{item.city}</Option>)}
+          <Input style={{ width: '40%' }} placeholder={t('home-search-name')} size="middle" onChange={(e: any) => setCraftsmanSearch(e.target.value)} />
```
Comment (devstm) [inline client/src/Components/WelcomeSearch/index.tsx:26]: but style inside css file

File: client/src/Components/WelcomeSearch/index.tsx [hunk 5]
```diff
@@ -14,19 +14,25 @@ interface ListOfLocation {
...
+          <Select defaultValue="بيت حانون" size="middle" onSelect={(citySearched: any) => setLocationSearch(citySearched)}>
```
Comment (devstm) [inline client/src/Components/WelcomeSearch/index.tsx:27]: i think defaultValue= all or choose city better than بيت حانون 
and what if it was a translated word? 
Isn't it better?

**Review by devstm — CHANGES_REQUESTED** [review]

File: client/src/Pages/Search.tsx (hunk 2 above)
Comment (RandSohail) [reply client/src/Pages/Search.tsx:11]: this text we don't need it at all just to show us which page we stand on {useTranslation().t('search-greeting')}

**Review by devstm — APPROVED** [review]

**Review by LinaYahya — APPROVED** [review]

**Pull request merged**